modules/images/dominant-color/load.php

```diff
@@ -112,12 +112,15 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 	}
 
 	$data        = '';
-	$style       = '';
 	$extra_class = '';
 
 	if ( ! empty( $image_meta['dominant_color'] ) ) {
 		$data .= sprintf( 'data-dominant-color="%s" ', esc_attr( $image_meta['dominant_color'] ) );
-		$style = 'style="--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';" ';
+		if ( str_contains( $filtered_image, ' style="' ) ) {
+			$filtered_image = str_replace( ' style="', ' style="--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';', $filtered_image );
+		} else {
+			$filtered_image = str_replace( '<img ', '<img style="--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';" ', $filtered_image );
+		}
```

Comment:
Can you please add a unit test for this change?

You could call `dominant_color_img_tag_add_dominant_color()` with an `img` tag that already has a `style` attribute to make sure it is combined correctly.

Reply:
@mukeshpanchal27 Excellent, thanks!